server/game/costs.js

```diff
@@ -43,6 +44,8 @@ const Costs = {
                     return true;
                 } else if (context.source.hasHouse(context.player.activeHouse)) {
                     return true;
+                } else if (context.source.hasKeyword('versatile')) {
```

Comment:
Same here:

```
this.abilities.keywordPersistentEffects.push(
            this.persistentEffect({
                printedAbility: false,
                condition: () => this.hasKeyword('versatile'),
                match: this,
                effect: AbilityDsl.effects.canUse(
                   (card, context, effectContext) => card === effectContext.source
                )
            })
        );
```

Very rough code, but the idea is to use existing effects.

Reply:
Done.